test-result-summary-client/src/Build/BuildDetail.jsx

```diff
@@ -110,6 +112,12 @@ export default class BuildDetail extends Component {
             />
             <br />
             <BuildTable title={"Children builds"} buildData={childBuildsDataSource} />
+            <br />
+            <Link to={{ pathname: '/releaseSummary', search: params({ parentId: parentId, buildName: buildName}) }}>
```

Comment:
<Link to={{ pathname: '/releaseSummary', search: params({ parentId, buildName}) }}>